src/sections/Community/Newcomers-guide/index.js

```diff
@@ -76,8 +78,8 @@ const NewcomersGuide = () => {
         </div>
         <div className="instructions">
           <Row className="step">
-            <Col sm={12} lg={6}>
-              <img src={CommunityImage1} className="finding-an-issue" alt="Community Image" />
+            <Col sm={12} lg={6} className="finding-an-issue" >
+              <StaticImage src={CommunityImage1} alt="Community Image" />
```

Comment:
This is coming out to be smaller than what we have currently. Can you try increasing the size slightly?

Reply:
okay

Reply:
![image](https://user-images.githubusercontent.com/64387054/152047133-59f4eada-5ba8-430b-88a1-6031dfb8c265.png)

@debo19 I tried many ways but was not able to manipulate its width, I thought `object-fit=contain` will work but it is also not working as imagined, can you guide me on how can I edit its width? do I need to edit something in `gastby-config` as I think some styles are also pre-defined there but I think it will affect other images as well because those are global styles defined for gatsby images of different sizes? `important` is also not working in this case.